ReactWindows/ChakraBridge/ChakraHost.h

```diff
@@ -2,6 +2,9 @@
 
 #include "pch.h"
 #include <jsrt.h>
+bool CompareLastWrite(const wchar_t* szPath1, const wchar_t* szPath2);
```

Comment:
This looks weird, any style suggestions for whitespace between #includes and method declarations?